localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -648,11 +649,11 @@ void EKFLocalizer::publishEstimateResult()
 void EKFLocalizer::updateSimple1DFilters(const geometry_msgs::msg::PoseWithCovarianceStamped & pose)
 {
   double z = pose.pose.pose.position.z;
-  double roll = 0.0, pitch = 0.0, yaw_tmp = 0.0;
 
-  tf2::Quaternion q_tf;
-  tf2::fromMsg(pose.pose.pose.orientation, q_tf);
-  tf2::Matrix3x3(q_tf).getRPY(roll, pitch, yaw_tmp);
+  const Eigen::Vector3d euler = quaternionToEulerXYZ(pose.pose.pose.orientation);
```

Comment:
Thank you for pointing this out :+1: 
I think it is better to use this: https://github.com/autowarefoundation/autoware.universe/blob/87641bff81a70b5b0df587a25903d800cc146163/common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp#L277-L283

Reply:
Resolved in [9aaa663](https://github.com/autowarefoundation/autoware.universe/pull/2696/commits/9aaa6630a9145a2b5eec93d2fbafee5340b294ff)